gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
```

If we're targeting 7.3, we should be doing this:

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```